panic during rebase

```
[master] git rebase --continue
[detached HEAD 6c9b82e] Prettify apis/hydra.json
 Author: prettifier[bot] <45367598+prettifier[bot]@users.noreply.github.com>
 1 file changed, 3043 insertions(+)
 create mode 100644 apis/hydra.json
Auto-merging _archive/website/versioned_docs/version-oryOS.17/hydra/configuration.md
CONFLICT (content): Merge conflict in _archive/website/versioned_docs/version-oryOS.17/hydra/configuration.md
error: could not apply 62763ff... Prettify docs/hydra/configuration.md
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 62763ff... Prettify docs/hydra/configuration.md
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x7c346d]

goroutine 1 [running]:
github.com/git-town/git-town/src/run.(*Result).Output(...)
        /home/kevlar/git-town/src/run/result.go:36
github.com/git-town/git-town/src/git.(*Runner).ContinueRebase(0xc0001343b8, 0x92c301, 0x0)
        /home/kevlar/git-town/src/git/runner.go:223 +0xbd
github.com/git-town/git-town/src/steps.(*ContinueRebaseBranchStep).Run(0xdf08a8, 0xc000134370, 0xa56520, 0xc0001d4620, 0xdc6a60, 0x7a5a09c1d108)
        /home/kevlar/git-town/src/steps/continue_rebase_branch_step.go:31 +0x5f
github.com/git-town/git-town/src/steps.Run(0xc00019ce00, 0xc000134370, 0xa56520, 0xc0001d4620, 0x9be1a0, 0xa56520)
        /home/kevlar/git-town/src/steps/run.go:43 +0x15f
github.com/git-town/git-town/src/cmd.glob..func13(0xdc0700, 0xdf08a8, 0x0, 0x0)
        /home/kevlar/git-town/src/cmd/continue.go:31 +0x16c
github.com/spf13/cobra.(*Command).execute(0xdc0700, 0xdf08a8, 0x0, 0x0, 0xdc0700, 0xdf08a8)
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:846 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0xdbbae0, 0x2, 0x1b, 0x0)
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:887
github.com/git-town/git-town/src/cmd.Execute()
        /home/kevlar/git-town/src/cmd/root.go:34 +0x71
main.main()
        /home/kevlar/git-town/main.go:10 +0x20
```

--- a/src/git/runner.go
+++ b/src/git/runner.go
@@ -218,9 +218,9 @@ func (r *Runner) ConnectTrackingBranch(name string) error {
 
 // ContinueRebase continues the currently ongoing rebase.
 func (r *Runner) ContinueRebase() error {
-	outcome, err := r.Run("git", "rebase", "--continue")
+	_, err := r.Run("git", "rebase", "--continue")
 	if err != nil {
-		return fmt.Errorf("cannot continue rebase: %w\n%s", err, outcome.Output())
+		return fmt.Errorf("cannot continue rebase: %w", err)
 	}
 	return nil
 }

